dev-python/catkin_pkg/catkin_pkg-0.4.16.ebuild

```diff
@@ -1,16 +1,17 @@
 EAPI=6
-PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+PYTHON_COMPAT=( python{2_7,3_5,3_6,3_7} )
 
 inherit distutils-r1
 
 DESCRIPTION="Standalone Python library for the Catkin package system."
 HOMEPAGE="http://wiki.ros.org/${PN}"
 SRC_URI="https://github.com/ros-infrastructure/${PN}/archive/${PV}.tar.gz -> ${P}-overlay.tar.gz"
 
-LICENSE="BSD-3"
+LICENSE="BSD"
```

Comment:
I don't know why that `BSD-3` string was there instead of `BSD` like most other packages. The LICENSE file in catkin_pkg just says BSD. By using BSD-3 emerge complains it needs a line in package.license to unmask it. So I thought the best solution was to just leave it as BSD instead, how it is in the original package.